packages/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.kt

```diff
@@ -107,6 +107,11 @@ class RNGestureHandlerModule(reactContext: ReactApplicationContext?) :
     val factory = RNGestureHandlerFactoryUtil.findFactoryForHandler<GestureHandler>(handler) ?: return
 
     factory.updateConfig(handler, config)
+
+    if (handler.actionType == GestureHandler.ACTION_TYPE_NATIVE_DETECTOR) {
+      interactionManager.dropRelationsForHandlerWithTag(handlerTag)
+      interactionManager.configureInteractions(handler, config)
+    }
```

Comment:
Shouldn't this be in `setGestureHandlerConfig`?

Reply:
As far as I remember, previous implementations rely only on setting the config, so yes, but that would need to be double-checked.